src/chrome/content/rules/Ruby-lang.org.xml

```diff
@@ -2,28 +2,34 @@
 	Nonfunctional subdomains:
 
 		- doc *
+		- redmine &
 
-	* Reset
+	* Presents cert for docs.ruby-lang.org
+	& Presents cert for bugs.ruby-lang.org
 
 
 	Fully covered subdomains:
 
 		- (www.)
 		- bugs
-		- redmine
+		- docs
 
 -->
-<ruleset name="ruby-lang.org (partial)">
+<ruleset name="ruby-lang.org">
 
 	<target host="ruby-lang.org" />
-	<target host="bugs.ruby-lang.org"/>
-	<target host="redmine.ruby-lang.org"/>
-	<target host="www.ruby-lang.org"/>
+	<target host="*.ruby-lang.org" />
```

Comment:
Right, if redirected blindly to HTTPS on the same domain, they will break on certificate mismatch. In my latest rebase, I chose to add specific redirects for those: redmine → bugs, doc → docs.

I also added the “See also” comments per your suggestion.